src/cmd/services/m3dbnode/config/config.go

```diff
@@ -260,7 +260,7 @@ type TickConfiguration struct {
 type BlockRetrievePolicy struct {
 	// FetchConcurrency is the concurrency to fetch blocks from disk. For
 	// spinning disks it is highly recommended to set this value to 1.
-	FetchConcurrency int `yaml:"fetchConcurrency" validate:"min=0"`
+	FetchConcurrency *int `yaml:"fetchConcurrency" validate:"min=1"`
```

Comment:
I believe this ignores the min: https://play.golang.org/p/0X5FrO3wuSp

Reply:
Looks to work, doesn't work in playground but here's what I was testing with:
https://play.golang.org/p/5MM-imeIsfn

And `foo.yaml`:
```yaml
internal: 0
```

And output:
```
err is unable to load config from [./foo.yaml]: Internal: less than min
```

Reply:
Ah cool, I tried it in the repo as well, but also with `yaml.Unmarshal` rather than with the config reader and it seemed to ignore there too

Reply:
Yeah we do a more strict yaml Unmarshal which runs the validator I believe. Plain old yaml.Unmarshal doesn't get you there, as you're pointing out 😆